contrib/tools/graphml/ngraph_json_to_graphml.py

```diff
@@ -0,0 +1,204 @@
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+# Simple converter to convert nGraph JSON format to GraphML.
+#  
+# Original Author: Vasanth Tovinkere
+# Date: 2/5/2018
+#
```

Comment:
This is more than a year old and is working with a reverse-engineered undocumented internal json serialization format  which has changed since then. There are no tests for this program.